### ISSUE TITLE
как программу полностью удалить? перезаписывает ли программа какие-нибудь системные или пользовательские файлы в /home? можно ли сделать flatpak чтобы можно было легко накатить на любой дистрибутив и легко удалить?

### ISSUE THREAD

**Issue by poleznosti** [bol-van/zapret#77]
я не спец в линуксе, поэтому вопросы возможно глупые, но даже странно что их не задавали.

**Comment by bol-van** [comment]
/opt/zapret/uninstall_easy.sh
rm -r /opt/zapret
концов быть не должно
другого деплоймента чем есть сейчас не будет

**Comment by poleznosti** [comment]
а почему не будет? во флатпаке что-нибудь сломается или там не получится работать с системой напрямую?

**Comment by bol-van** [comment]
потому что я ставлю себе задачу создать средство для рубящих в теме а не для домохозяек. тема достаточно сложная раз два нажал заработало обычно не бывает нужна настройка под провайдера

**Comment by bol-van** [comment]
на счет концов   на опенврт доставляется куча ненужных в ином случае пакетов. uninstall их не сносит и тем свмыи значительно сьедается место

**Comment by poleznosti** [comment]
да, действительно, сейчас погуглил и выяснилось что в /opt писать на сильверблю можно, вот ссылка для тех кто возможно будет задаваться тем же вопросом https://docs.fedoraproject.org/en-US/fedora-silverblue/technical-information/

> On Silverblue, the root filesystem is immutable. This means that /, /usr and everything below it is read-only.
> 
> /var is where all of Silverblue’s runtime state is stored. Symlinks are used to make traditional state-carrying directories available in their expected locations. This includes:
> 
> /home → /var/home
> /opt → /var/opt
> /srv → /var/srv
> /root → /var/roothome
> /usr/local → /var/usrlocal
> /mnt→ /var/mnt
> /tmp → /sysroot/tmp
> 
> This means that separate home partitions should be mounted on /var/home.

**Comment by bol-van** [comment]
Если opt представляет собой symbolic link на другую rw локацию - это нормально
У самого на openwrt  /opt => /home/opt, и все нюансы, с этим связанные, учтены
Но простой установщик устанавливает systemd юниты в /lib/systemd/system, а это уже точно не прокатит
Может еще что-то
рекомендую идти ручным режимом. самому разобраться как запустить нужный режим через sh скрипт (iptables + nfqws/tpws) и как сделать его автозагрузку

**Comment by bol-van** [comment]
Или можно использовать и мои скрипты, но запускать это как-то так
/opt/zapret/init.d/sysv/zapret start
естественно, предварительно настроить /opt/zapret/config вручную

**Comment by poleznosti** [comment]
да, в этом будет тяжко разбираться. жаль что нет флатпака или хотя бы rpm, тогда бы это всё можно было бы встроить в систему банальным "rpm-ostree zapret.rpm".

**Comment by bol-van** [comment]
Я бы не назвал предложенное решение слишком сложным
вручную скопипастить 1 текстовый файл, сделать ему chmod 755, выполнить install_easy.sh, отключить firewalld
что-то из этих действий непонятно ?
